packages/roosterjs-editor-api/lib/format/createLink.ts

```diff
@@ -67,6 +67,7 @@ export default function createLink(
 
         editor.addUndoSnapshot(() => {
             let range = editor.getSelectionRange();
+

```

Comment:
no need to add empty line here

Reply:
Can I ask what is the criteria for when an emty line is allowed and when it is not?